tasks/document-ocr-fn/resources/function/ocr.py

```diff
@@ -15,29 +17,45 @@ def trOCR(image: Image.Image) -> str:
     return processor.batch_decode(generatedIds, skip_special_tokens = True)[0]  # type: ignore
 
 
+def parseDate(inputDate: str) -> dict[str, Optional[int]]:
+    # Replace all periods with spaces
+    inputDate = inputDate.replace(".", " ")
+
+    # "DD MM YY" -> "DD MM YYYY" [old Danish passorts]
+    if len(inputDate.split(" ", 3)[2]) == 2:
```

Comment:
What if this raises `IndexError`?

Reply:
I will add a try except block around it and return None for day, month and year DOB fields for now. If any specific exceptions that can be preprocessed are encountered in the future, they can be added then.